python/PiFinder/keyboard.py

```diff
@@ -6,7 +6,10 @@
 
 """
 import sh
```

Comment:
This would be another great place to use a 'fake' module.  There could be a keyboard_fake.py structured the same way which collects keystrokes (via pygame?) and feeds them in to the queue back to main.py

Reply:
indeed, should have a look how the whole queue things work

Reply:
done